### ISSUE TITLE
Stops working when screen turned off and then back on

### ISSUE THREAD

**Issue by Gitoffthelawn** [takke/DataStats#15]
Steps I took:
1. Install DataStats ( https://github.com/takke/DataStats )
2. Set DataStats to auto start on boot
3. Reboot*
4. Install WiFi-Automatic ( https://github.com/j4velin/WiFi-Automatic )
5. Reboot
6. Let system stabilize after boot
6. Turn off screen
7. Wait a few seconds
8. Turn on screen
9. DataStats is now frozen at whatever level it was displaying when screen turned off

Now the weird part: I uninstalled WiFi-Automatic and rebooted, and the same symptoms persisted.

I have posted an Issue on the WiFi-Automatic github to see if it is possible that WiFi-Automatic changes something that even after it is uninstalled could affect DataStats?

Before installing WiFi-Automatic, I had been using DataStats without an issue for quite some time.

So what else was changed?  Notice the asterisk at Step # 3 above.  When I rebooted, I manually updated a couple apps (I wasn't trying to run an experiment) and ran (it was already installed) and then uninstalled Wifi Key Recovery ( https://github.com/alt236/Wifi-Key-Recovery---Android ).

I looked at the updated apps, but I don't think they are relevant to the issue.  Since the symptom seems to directly related to the functionality of WiFi-Automatic, I think it's worth checking to see if it could be a part of the problem.  Running Wifi Key Recovery could also be worth seeing if it creates a conflict.

The bottom line is that DataStats is suddenly unusable, as it stops working after every time the screen is cycled off and then back on.

**Comment by Gitoffthelawn** [comment]
Wow, this issue sure is a blast from the past! ;)
